cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -120,6 +123,54 @@ function ManagementCredentials(props: Partial<StepProps>) {
             }
         }
     };
+
+    const isConnected = () => {
+        if (connectionStatus === CONNECTION_STATUS.CONNECTED) {
+            return true;
+        }
+        return false;
+    };
+
+    const hasError = () => {
+        if (Object.keys(errorObject).length > 0) {
```

Comment:
Likewise, I think it may be simpler here to just `return Object.keys(errorObject).length > 0`

Reply:
Yes sir. Done